missioncontrol/etl/measure.py

```diff
@@ -113,14 +102,46 @@ def update_measures(application_name, platform_name, channel_name,
         'os_name': platform.telemetry_name,
         'channel_name': channel_name,
         'min_timestamp': min_timestamp.strftime("%Y-%m-%d %H:%M:%S"),
-        'submission_date': submission_date.strftime("%Y%m%d")
+        'submission_date': submission_date.strftime("%Y-%m-%d")
     }
-    logger.info('Querying: %s', query_template % params)
+
+    bq_query = f'''
+    SELECT 
+        window_start,
+        build_id, 
+        display_version, 
+        SUM(usage_hours) summed_usage_hours,
+        SUM(count),
+        {measure_sums}
+    FROM 
+        {MISSION_CONTROL_TABLE}
+    WHERE
+        submission_date = \'{submission_date.strftime('%Y-%m-%d')}\'
+        AND application = \'{params['application_name']}\'
+        AND display_version > \'{params['min_version']}\'
+        AND display_version < \'{params['max_version']}\'
+        AND build_id > \'{params['min_build_id']}\'
+        AND build_id < \'{params['max_build_id']}\'
+        AND os_name = \'{params['os_name']}\'
+        AND channel = \'{params['channel_name']}\'
+        AND window_start > \'{params['min_timestamp']}\'
+    GROUP BY
+        window_start, 
+        build_id, 
+        display_version
+    HAVING
+        summed_usage_hours > 0
+    '''.replace('\n', '').strip()
+
+    logger.info('Querying: %s', bq_query)
+
+    client = get_bigquery_client()
+    query_job = client.query(query=bq_query)
 
     # bulk create any new datum objects from the returned results
     build_cache = {}
     datum_objs = []
-    for row in raw_query(query_template, params):
+    for row in query_job:
```

Comment:
Shouldn't this be `query_job.result()`? https://googleapis.dev/python/bigquery/latest/index.html#perform-a-query

Reply:
Iterating over the job is possible too: https://cloud.google.com/bigquery/docs/reference/libraries#client-libraries-usage-python.  I think result is implicitly called here.  Do you have any readability preference for this?

Reply:
Ah, ok, I think the way you wrote it is fine in that case. Just Google's documentation threw me off.